lib/helper/scripts/isElementClickable.js

```diff
@@ -0,0 +1,25 @@
+function isElementClickable(elem) {
+  if (!elem.getBoundingClientRect || !elem.scrollIntoView || !document.elementFromPoint) {
+    return false;
+  }
+
+  const isElementInViewport = (elem) => {
+    const rect = elem.getBoundingClientRect();
+    const verticleInView = (rect.top <= window.innerHeight) && ((rect.top + rect.height) > 0);
+    const horizontalInView = (rect.left <= window.innerWidth) && ((rect.left + rect.width) > 0);
+    return horizontalInView && verticleInView;
+  };
+
+  const getOverlappingElement = (elem) => {
+    const rect = elem.getBoundingClientRect();
+    const x = rect.left + (elem.clientWidth / 2);
+    const y = rect.top + (elem.clientHeight / 2);
+    return document.elementFromPoint(x, y);
+  };
+
+  const isClickable = elem => elem.disabled !== true && isElementInViewport(elem) && getOverlappingElement(elem) === elem;
+  return isClickable(elem);
+}
+
+module.exports = isElementClickable;
+
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Too many blank lines at the end of file. Max of 0 allowed. (no-multiple-empty-lines)](https://app.codacy.com/app/DavertMik/CodeceptJS/pullRequest?prid=4489601)